st-embroider/2023/2023-12-19.md

```diff
@@ -0,0 +1,79 @@
+# 2023-12-19
+
+Note Taker: Preston
+Time Keeper: Andrey Mikhaylov
+
+## Attendees
+
+Add yourself to the list if you attend and check the box!
+
+- [x] Ed Falkner (EF)
+- [x] Chris Manson (CM)
+- [ ] Aaron Chambers (AC)
+- [x] Alex (A)
+- [x] Preston Sego (PS)
+- [ ] Simon Ihmig (SI)
+- [x] Katie Gengler (KG)
+- [ ] Peter Meehan (PM)
+- [x] Andrey Mikhaylov (AM)
+- [ ] Godfrey Chan (GC)
+- [x] Tommy JR (TJ)
+- [ ] heroiceric
+- [ ] Nick Schot (NS)
+- [x] David Taylor 
+- [x] Peter Wagenet
+- [x] Krystan
+- [x] Ignace 
+
+## Topics
+
+### content-tag indi
+
+https://github.com/embroider-build/content-tag/issues/45 
+
+- content-tag _is_ byte ranges
+- will add APIs to content-tag so that users don't need to do slicing
+  - https://github.com/embroider-build/content-tag/issues/39#issuecomment-1832443310
+- potentially add to content-tag:
+  - charStart, charEnd, etc
+  - but wouldn't be needed if there is a `transform` callback capability
+- Krystan volunteers to PR documentation
+
+### GH actions magically turned on, on forks and unknowingly almost published a new unstable when sync-ing the fork 
+
+- when forking, if users have turned actions on by default for their profile/org, they'll need to manually turn actions off for that fork
+- GH problem, GH needs more controls over this
+
+### v2 addons gjs/gts module name exposing system config and bloating bundle size 
+
+- production path includes full system path (sometimes over 300 characters)
+- we want to remove the module name altogether because module name doesn't make sense for `<template>` components.
+- add setting to remove?
+  - potentially in babel-plugin-ember-template-compilation
+- RFC to remove `moduleName`
+- TODO: investigate -- does ember-intl use `moduleName`? we know the inspector does
+  - ember language server's ember-intl integration?
+- Idea: `<template>` doesn't emit `moduleName`
+  - need to communicate that `moduleName` is optional / sometimes missing
```

Comment:
This is already the case – `precompile` has `moudleName` marked as optional, technically, `moduleName` on the template is always present, but only because we default it to this string https://github.com/glimmerjs/glimmer-vm/blob/cb867e388d0de3547b3f4ebc10e2e92c95b75d7e/packages/%40glimmer/compiler/lib/compiler.ts#L128

So it would be non-breaking, at least technically speaking, to always set it to that string, and consumers should already be accounting for that